Tasks/SonarQubePostTest/SonarQubePostTestImpl.ps1

```diff
@@ -0,0 +1,106 @@
+function InvokeMsBuildRunnerPostTest
+{
+	$bootstrapperPath = GetBootsrapperPath
+	$arguments = GetMsBuildRunnerPostTestArgs
+
+	Invoke-BatchScript $bootstrapperPath -Arguments $arguments
+}
+
+function GetBootsrapperPath
+{
+	$bootstrapperPath = GetTaskContextVariable "MsBuild.SonarQube.BootstrapperPath" 
+
+	if (!$bootstrapperPath -or ![System.IO.File]::Exists($bootstrapperPath))
```

Comment:
The most likely cause for the first error is that the pre-build step is missing, which is what the error message says.
However, if the property is set but the exe can't be found then it is probably something else (probably more serious, such as an error in the tasks/mis-matched set of tasks). For the second condition, it might be useful to dump the path that couldn't be found to the output.